pysonnet/projects.py

```diff
@@ -376,6 +376,25 @@ class GeometryProject(Project):
     """
     Class for creating and manipulating a Sonnet geometry project.
     """
+    
+    # exports current data as a .csv file
+   def export_command(sonnet_path,xml,sonnet_file):
```

Comment:
- let's move this function below the `make_sonnet_file()` method.
- self is always the first argument to a class method
- we need to programmatically generate the xml file for this conversion inside of this function. (remove it from the arguments)
- remove the sonnet_path variable. Use self['sonnet']["sonnet_path"] instead
- use self.project_file_path for the sonnet_file variable. Catch the case where it hasn't been set and raise an error
- let's use a more descriptive function name like `export_current_density`